src/frontend/sharedComponents/CustomHeaderLeftClose.tsx

```diff
@@ -68,51 +85,80 @@ export const CustomHeaderLeftClose = ({
   headerBackButtonProps,
   observationId,
 }: CustomHeaderLeftCloseProps) => {
-  if (observationId) {
-    return (
-      <HeaderBackEditObservation
-        tintColor={tintColor}
-        headerBackButtonProps={headerBackButtonProps}
-        observationId={observationId}
-      />
+  const {isOpen, sheetRef, closeSheet, openSheet} = useBottomSheetModal({
+    openOnMount: false,
+  });
+  const {formatMessage} = useIntl();
+  const {clearDraft} = useDraftObservation();
+  const navigation = useNavigationFromRoot();
+
+  const handleDiscard = React.useCallback(() => {
+    clearDraft();
+    navigation.dispatch(
+      CommonActions.reset({index: 0, routes: [{name: 'Home'}]}),
     );
-  }
+  }, [clearDraft, navigation]);
 
   return (
-    <HeaderBackNewObservation
-      tintColor={tintColor}
-      headerBackButtonProps={headerBackButtonProps}
-    />
+    <>
+      {observationId ? (
+        <HeaderBackEditObservation
+          tintColor={tintColor}
+          headerBackButtonProps={headerBackButtonProps}
+          observationId={observationId}
+          openBottomSheet={openSheet}
+        />
+      ) : (
+        <HeaderBackNewObservation
+          tintColor={tintColor}
+          headerBackButtonProps={headerBackButtonProps}
+          openBottomSheet={openSheet}
+        />
+      )}
+      <BottomSheetModal isOpen={isOpen} ref={sheetRef}>
+        <BottomSheetContent
+          title={
+            observationId
+              ? formatMessage(m.discardChangesTitle)
+              : formatMessage(m.discardTitle)
+          }
+          description={
+            observationId
+              ? formatMessage(m.discardChangesDescription)
+              : formatMessage(m.discardObservationDescription)
+          }
+          buttonConfigs={[
+            {
+              variation: 'filled',
+              dangerous: true,
+              onPress: handleDiscard,
+              text: observationId
+                ? formatMessage(m.discardChangesButton)
+                : formatMessage(m.discardObservationButton),
+              icon: <DiscardIcon />,
+            },
+            {
+              onPress: closeSheet,
+              text: formatMessage(m.discardCancel),
```

Comment:
nit: Seeing the cancel action copy of `Continue editing` seems a little off when you're on the category chooser screen.

Reply:
Im going to leave this because we are planning on updating all the copy at a later time, so that can be dealt with then